Source/RunActivity/Viewer3D/Debugging/TimetableWindow.cs

```diff
@@ -0,0 +1,928 @@
+﻿// COPYRIGHT 2010 - 2020 by the Open Rails project.
+// 
+// This file is part of Open Rails.
+// 
+// Open Rails is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// Open Rails is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with Open Rails.  If not, see <http://www.gnu.org/licenses/>.
+
+using Microsoft.Xna.Framework;
+using Orts.Formats.Msts;
+using Orts.MultiPlayer;
+using Orts.Simulation;
+using Orts.Simulation.Physics;
+using Orts.Simulation.RollingStocks;
+using ORTS.Common;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Drawing;
+using System.Linq;
+
+namespace Orts.Viewer3D.Debugging
+{
+    public class TimetableWindow
+    {
+        public DispatchViewer F { get; set; } // Shortest possible abbreviation so code is easier to read.
+
+        public TimetableWindow(DispatchViewer form)
+        {
+            F = form;
+        }
+
+		public void SetControls()
```

Comment:
Will do

Reply:
I believe it is because there are indents using tabs, and indents using spaces. GitHub uses 8 spaces per a tab, while VS uses 4 only, that's why it isn't visible in VS.